{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -290,39 +290,97 @@
     'filters': {
         'require_debug_false': {
             '()': 'django.utils.log.RequireDebugFalse'
-        }
+        },
+        'require_debug_true': {
+            '()': 'django.utils.log.RequireDebugTrue',
+        },
     },
     'formatters': {
+        'simple': {
+            'format': '%(levelname)s %(message)s'
+        },
         'verbose': {
             'format': '%(levelname)s %(asctime)s %(module)s '
                       '%(process)d %(thread)d %(message)s'
         },
     },
     'handlers': {
-        'mail_admins': {
-            'level': 'ERROR',
-            'filters': ['require_debug_false'],
-            'class': 'django.utils.log.AdminEmailHandler'
+        'null': {
+            'class': 'logging.NullHandler',
         },
         'console': {
             'level': 'DEBUG',
+            'filters': ['require_debug_true'],
+            'formatter': 'simple',
             'class': 'logging.StreamHandler',
+        },
+        'debug': {
+            'level': 'DEBUG',
+            'filters': ['require_debug_false'],
+            'formatter': 'verbose',
+            'class': 'logging.NullHandler',
+            'filename': '/var/log/netdoktor/nd/django_debug.log',
+            'when': 'midnight',
+            'interval': 1,
+            'backupCount': 7,
+        },
+        'error': {
+            'level': 'ERROR',
+            'filters': ['require_debug_false'],
             'formatter': 'verbose',
+            'class': 'logging.NullHandler',
+            'filename': '/var/log/netdoktor/nd/django_error.log',
+            'when': 'midnight',
+            'interval': 1,
+            'backupCount': 31,
         },
+        'mail_admins': {
+            'level': 'ERROR',
+            'filters': ['require_debug_false'],
+            'formatter': 'verbose',
+            'class': 'django.utils.log.AdminEmailHandler',
+        }
     },
     'loggers': {
         'django.request': {
-            'handlers': ['mail_admins'],
+            'handlers': ['console', 'mail_admins'],
             'level': 'ERROR',
             'propagate': True
         },
-        'django.security.DisallowedHost': {
-            'level': 'ERROR',
+        'django.security': {
+            'level': 'WARNINGS',
             'handlers': ['console', 'mail_admins'],
             'propagate': True
         }
+        '': {
+            'handlers': ['console', 'debug', 'error'],
+            'propagate': True,
+        },
     }
 }
+
+# Log debug statements into DJANGO_DEBUG_LOG_FILE
+# with daily rotation at midnight and backup of 1 week
+if env('DJANGO_DEBUG_LOG_FILE'):    
```

Comment:
please, add `DJANGO_DEBUG_LOG_FILE` environment variable on this documentation:
https://github.com/pydanny/cookiecutter-django/blob/master/docs/settings.rst